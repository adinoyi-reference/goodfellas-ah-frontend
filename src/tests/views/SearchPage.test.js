import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchPage from '../../views/SearchArticles';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<SearchPage />);
});

describe('SearchPage UI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
