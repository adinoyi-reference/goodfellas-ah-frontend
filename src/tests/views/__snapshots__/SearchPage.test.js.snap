// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchPage UI render features view should render as expected 1`] = `
<div>
  <Connect(Header) />
  <Connect(SearchArticlesContainer) />
</div>
`;
